common/content-api.js

```diff
@@ -250,10 +275,7 @@ function getMerchandise(locale, showAll = false) {
 }
 
 module.exports = {
-    mapAttrs,
```

Comment:
`mapAttrs` and `mergeWelshBy` are never used outside of this file so we can avoid the awkard export comment